apps/openchallenges/organization-service/src/test/java/org/sagebionetworks/openchallenges/organization/service/model/entity/OrganizationEntityTest.java

```diff
@@ -0,0 +1,227 @@
+package org.sagebionetworks.openchallenges.organization.service.model.entity;
+
+import java.time.OffsetDateTime;
+import java.util.Collections;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class OrganizationEntityTest {
+
+  @Test
+  public void
+      OrganizationEntityBuilderArguments_ShouldReturnArgumentsPassed_WhenOrganizationEntityBuilt() {
+
+    Long id = 1L;
+    String name = "Test Organization";
+    String email = "test@example.com";
+    String login = "testorg";
+    String avatarKey = "avatarKey";
+    String websiteUrl = "https://example.com";
+    Integer challengeCount = 5;
+    String description = "Test description";
+    OffsetDateTime createdAt = OffsetDateTime.now();
+    OffsetDateTime updatedAt = OffsetDateTime.now();
+    String acronym = "TO";
+
+    OrganizationEntity organizationEntity =
+        OrganizationEntity.builder()
+            .id(id)
+            .name(name)
+            .email(email)
+            .login(login)
+            .avatarKey(avatarKey)
+            .websiteUrl(websiteUrl)
+            .challengeCount(challengeCount)
+            .categories(Collections.emptyList())
+            .challengeContributions(Collections.emptyList())
+            .description(description)
+            .createdAt(createdAt)
+            .updatedAt(updatedAt)
+            .acronym(acronym)
+            .build();
+    Long retrievedId = organizationEntity.getId();
```

Comment:
Two name terms used often together are "actual" and "expected" value. Here you could replace "retrieved" by "actual". You could optionally define above "expectedName", etc. but it's fine with me to keep using "name", "email", etc.

As usual, please consider other places where this comment may apply (I may occasionally no longer mention this).